keycloak_backend/api/hipcloak.py

```diff
@@ -234,12 +234,12 @@ def get_group_details_from_path(self, group_path):
             If the group does not exist, it prints a message and returns None.
         """
         try:
-            group_info = self._kc_admin.get_group_by_path(group_path)
-            if group_info:
-                return group_info
-            else:
+            group = self._kc_admin.get_group_by_path(group_path)
+            if not group:
                 print(f"Group not found at path '{group_path}'.")
-                return None
+                return
+
+            return self._kc_admin.get_group(group["id"])
```

Comment:
the `subGroups` list is not populated.

https://www.keycloak.org/docs/23.0.7/upgrading/#grouprepresentation-changes